task/s2i/0.2/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
```

Comment:
can we switch from centos:7 (which is deprecated) to ubi which is much smaller too?